### PR TITLE
Align RequestURI attribute creation in Jetty 12 with Jetty 11 approach.

### PR DESCRIPTION
**Pull request by sondemar**
closes https://github.com/qos-ch/logback-access/issues/4

### CODE REVIEW

File: jetty12/src/test/java/ch/qos/logback/access/jetty/SiftingAppenderTest.java [hunk 1]
```diff
@@ -66,15 +66,15 @@ public void invokingDifferentPathShouldBeSiftedAccordingly() throws Exception {
         Set<String> keySet = siftingAppender.getAppenderTracker().allKeys();
         assertEquals(3, keySet.size());
 
-        Set<String> witnessSet = new LinkedHashSet<String>();
-        witnessSet.add(prefix);
-        witnessSet.add(prefix+"x");
-        witnessSet.add(prefix+"y");
+        Set<String> witnessSet = new LinkedHashSet<>();
```
Comment (sondemar) [inline jetty12/src/test/java/ch/qos/logback/access/jetty/SiftingAppenderTest.java:69]: compliant with https://github.com/sondemar/logback-access/blob/0e1f57a69891df361e221a82f083619be6b97451/jetty11/src/test/java/ch/qos/logback/access/jetty/SiftingAppenderTest.java#L53

**Comment by ceki** [comment]
@sondemar Thank you for this PR. It fails DCO check. You should be able to add the proper sign-off with the help of the instructions in the DCO check link above.

**Comment by sondemar** [comment]
> @sondemar Thank you for this PR. It fails DCO check. You should be able to add the proper sign-off with the help of the instructions in the DCO check link above.

@ceki , now it should be fine

**Comment by ceki** [comment]
@sondemar  Logback-access version 2.0.1 containing this fix has been released earlier today.

**Comment by sondemar** [comment]
> @sondemar Logback-access version 2.0.1 containing this fix has been released earlier today.

Thank you @ceki

**Pull request merged**